README.rst

```diff
@@ -329,7 +329,7 @@ UNIX systems
 **BSD**:
 
 - OpenBSD (``pip`` installation)
-- FreeBSD 9/10/11
+- FreeBSD 9/10/11/12
```

Comment:
I guess the releases 9 and 10 would not be supported for real...
Need to adjust deprecation warnings in the script, but that is for another PR.